test/src/org/labkey/test/tests/cds/CDSGroupTest.java

```diff
@@ -187,7 +188,9 @@ public void verifyGroups()
 
         CDSHelper.NavigationLink.HOME.makeNavigationSelection(this);
         waitForText(STUDY_GROUP);
-        click(Locator.tagWithClass("div", "grouplabel").withText(STUDY_GROUP));
+        WebElement groupLabel = Locator.tagWithClass("div", "grouplabel").withText(STUDY_GROUP).findElement(getDriver());
+        shortWait().until(ExpectedConditions.elementToBeClickable(groupLabel));
+        groupLabel.click();
```

Comment:
FYI. This could be a simplified a bit.
```suggestion
        shortWait().until(ExpectedConditions.elementToBeClickable(
            Locator.tagWithClass("div", "grouplabel").withText(STUDY_GROUP)))
            .click();
```